src/core/aio/sim_aio_provider.h

```diff
@@ -24,22 +24,11 @@
  * THE SOFTWARE.
  */
 
-/*
- * Description:
- *     What is this file about?
- *
- * Revision history:
- *     xxxx-xx-xx, author, first version
- *     xxxx-xx-xx, author, fix bug about xxx
- */
-
 #pragma once
 
-#include <dsn/tool_api.h>
-#include "core/tools/common/native_aio_provider.linux.h"
+#include "native_linux_aio_provider.h"
 
 namespace dsn {
-namespace tools {
```

Comment:
I think a 'dsn::aio::' namespace is better.

Reply:
Maybe we can do it in later pull request. Because too many files are involved if we change namespace `dsn::` to `dsn::aio::`. And it is not the purpose of this pull request